samples/azure/azure-service-bus-msmq-bridge/ASB_8/Bridge/Program.cs

```diff
@@ -45,11 +45,11 @@ class Program
 
         // TODO: Bridge 2.x doesn't support this yet
 
-        bridgeConfiguration.InterceptForwarding(async (queue, message, forward) =>
+        bridgeConfiguration.InterceptForwarding(async (queue, message, dispatch, forward) =>
         {
             using (new TransactionScope(TransactionScopeOption.Suppress, TransactionScopeAsyncFlowOption.Enabled))
             {
-                await forward().ConfigureAwait(false);
+                await forward(dispatch).ConfigureAwait(false);
```

Comment:
@SzymonPobiega the comment above is no longer relevant then, right?

Reply:
So is the comment about serializer above
```
// TODO: ASB requires serializer to be registered.
         // Currently, there's no way to specify serialization for the bridged endpoints		         // Currently, there's no way to specify serialization for the bridged endpoints
         // endpointConfiguration.UseSerialization<T>();		         // endpointConfiguration.UseSerialization<T>();
```

Reply:
I removed `// TODO: Bridge 2.x doesn't support this yet` comments but the serializer seems to still be a problem in ASB

Reply:
> serializer seems to still be a problem in ASB

ASB or ASQ. Both require the default serializer to be set.
For ASB a fake one can be registered though.